app/models/concerns/database_indexable.rb

```diff
@@ -42,4 +42,10 @@ def generate_searchable_content # rubocop:disable Metrics/AbcSize,Metrics/Method
       ],
     ).tsvector
   end
+
+  private
+
+  def deduplicate(text)
+    text.squeeze(" ").gsub(/[^\w\s]/, "").downcase.split.uniq.join(" ")
```

Comment:
You don't have to join at the end - `TsvectorGenerator` will join arrays internally! Maybe this can then be called `unique_words`?

Reply:
👍

Reply:
I also forgot `#scan` exists.